server/plugin.go

```diff
@@ -513,6 +535,25 @@ func (p *Plugin) onActivate() error {
 			return dbErr
 		}
 
+		// simulate a ping to the plugin from server so it appears "online" immediately.
+		// !!!! remove this when fixed in server
+		_, err = db.Exec("update remoteclusters set lastpingat=$1 where remoteid=$2;", model.GetMillis(), p.remoteID)
+		if err != nil {
+			return fmt.Errorf("cannot simulate ping: %w", err)
+		}
+
+		// writing the invite directly in the DB
+		// !!!! remove this when fixed in server
+		p.inviteRemoteToChannel = func(channelID, remoteID, userID string) error {
+			now := model.GetMillis()
+			_, err2 := db.Exec("INSERT into sharedchannelremotes (id, channelid, creatorid, createat, updateat, isinviteaccepted, isinviteconfirmed, remoteid, lastpostupdateat, lastpostid, lastpostcreateat) values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11) ON CONFLICT (channelid, remoteid) DO NOTHING", model.NewId(), channelID, userID, now, now, true, true, remoteID, now, "", now)
+			if err2 != nil {
+				p.API.LogError("cannot simulate invite", "error", err2)
```

Comment:
Should we log this as a warning, or will a human be asked to intervene when this log is emitted?

Reply:
Good call, it should be a warning.